exotations/solvers/exotica_ilqr_solver/init/ilqr_solver.in

```diff
@@ -0,0 +1,6 @@
+class ILQRSolver
+
+extend <exotica_core/motion_solver>
+Optional int Patience = 10; // early stopping or patience
+Optional double ConvergenceTolerance = 1e-3; 
```

Comment:
In other solvers we sometimes call this `FunctionTolerance` and `FunctionTolerancePatience` if the relative function tolerance is meant.